syft/frameworks/torch/mpc/primitives.py

```diff
@@ -191,3 +237,34 @@ def build_separate_fss_keys(n_party: int, n_instances: int = 100):
             return [((alpha - mask) % 2 ** n, s_00, *CW), (mask, s_01, *CW)]
 
         return build_separate_fss_keys
+
+    def build_triples(self, op: str):
+        """
+        The builder to generate beaver triple for multiplication or matrix multiplication
+        """
+
+        def build_separate_triples(n_party: int, n_instances: int, **kwargs):
+            assert n_party == 2, (
+                "Only 2 workers supported for the moment. "
+                "Please fill an issue if you have an urgent need."
+            )
+            shapes = kwargs["shapes"]
+            if not isinstance(shapes, list):
+                assert len(shapes) == 2
```

Comment:
Q: Shouldn't this assert be after the ```if```?

Reply:
not really: shapes should actually be a list of pair of shapes. Will make it clear in the code